File: from-1601-to-1700/1614-maximum-nesting-depth-of-the-parentheses/solution.go
func maxDepth(s string) int {
    result, deep := 0, 0
    for i := 0; i < len(s); i ++ {
        if s[i] == '(' {
            deep += 1
            if deep > result {
                result = deep
            } 
        } else if s[i] == ')' {
            deep -= 1
        }
    }
    return result
}

File: from-1601-to-1700/1614-maximum-nesting-depth-of-the-parentheses/solution.py
class Solution:
    def maxDepth(self, s: str) -> int:
        ans = 0
        stack = 0
        for c in s:
            if c == '(':
                stack += 1
                ans = max(ans, stack)
            elif c == ')':
                stack -= 1
                
        return ans